staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -930,6 +924,8 @@ func decode(codec runtime.Codec, versioner storage.Versioner, value []byte, objP
 		} else {
 			klog.Infof("Could not set ClusterName %s in appendListItem on object: %T", clusterName, objPtr)
 		}
+	} else {
+		klog.Errorf("Cluster should not be unknown")
```

Comment:
Should this return an error instead of just logging it?

(If so, same comments elsewhere)

Reply:
Missed the comments about this above, retracting.